pywr/recorders/_recorders.pyx

```diff
@@ -1438,10 +1438,10 @@ cdef class RollingMeanFlowNodeRecorder(NodeRecorder):
         else:
             self.days = 0
         self._data = None
+        self.position = 0
 
     cpdef setup(self):
         super(RollingMeanFlowNodeRecorder, self).setup()
-        self.position = 0
```

Comment:
`RollingWindowParameterRecorder` initialises `position` in `setup`. I don't thinks this makes any real difference but it might be good to have them consistent?

Reply:
It does it in `reset` too?